src/importexport/capella/internal/capxml.cpp

```diff
@@ -669,7 +669,7 @@ void TrillObj::readCapx(XmlReader& e)
 void WedgeObj::readCapx(XmlReader& e)
 {
     // TODO: read LineObj properties
-    decresc          = e.asciiAttribute("decrescendo", "false") == "true";
+    dim              = e.asciiAttribute("decrescendo", "false") == "true"; // this is what Capella calls it internally
```

Comment:
Let's not rename this one, as it seems to be intended to reflect the Capella terminology

Reply:
done